editor/animation_track_editor.cpp

```diff
@@ -4989,7 +4987,9 @@ void AnimationTrackEditor::_update_step(double p_new_step) {
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
 	undo_redo->create_action(TTR("Change Animation Step"));
-	float step_value = p_new_step;
+	// To ensure that the conversion results are consistent between serialization and load, the value is snapped with 0.0625 to be a rational number.
+	// step_value must also be less than or equal to 1000 to ensure that no error accumulates due to interactions with retrieving values from inner range.
+	double step_value = fmin(1000.0, Math::snapped(p_new_step, FACTOR));
```

Comment:
```suggestion
	double step_value = Math::min(1000.0f, Math::snapped(p_new_step, FACTOR));
```
Math class should be used for consistency within Godot.

Reply:
I would but....
![image](https://github.com/user-attachments/assets/5ed0170e-7fd2-4916-b652-be51d299df9b)
So this would need to be added to `math_funcs.h` since `min` doesn't appear to be defined in Math in the v4.4dev branch

Reply:
Oh sorry I mistake, then you can use `MIN()`.

Reply:
Ah, that solves that then

Reply:
committed fixes, resolving